### ISSUE TITLE
Multi-Page support

### ISSUE THREAD

**Issue by eMPee584** [w3c/svgwg#157]
This has come up at least twice on the mailing list in recent years:
http://lists.w3.org/Archives/Public/www-svg/2013Mar/0064.html
http://lists.w3.org/Archives/Public/www-svg/2014Oct/0023.html

A decade back, a concept had been drafted for SVG1.2, which later got dropped.
https://www.w3.org/TR/2004/WD-SVG12-20041027/multipage.html

From a user standpoint, this is mainly useful in creating paginated documents for print (or print-like web publication). For my favorite tool inkscape, an extension exists (http://inkscape-pages.sourceforge.net) which implements this functionality by using layers.

Would it be sensible to standardize that approach? :thought_balloon: …

**Comment by jarek-foksa** [comment]
The old proposal would be best suited for authoring large multi-page documents such as books, which in my opinion should not be the purpose of SVG.

I would opt for a simpler and more flexible concept of multiple artboards, where each artboard could be printed as a separate page. Perhaps  `<view>` element could be extended to facilitate this?

**Comment by tabatkins** [comment]
Yeah, hints to the UA about the image pieces being "separate" and thus appropriate to print on separate pages would probably be useful.  Support for multi-page "documents" is out-of-scope for SVG; that's what HTML's for.

**Comment by eMPee584** [comment]
I agree that for _large_ (and especially structured) documents, SVG is probably not the adequate tool.
But no pagination support at all? What about multi-page booklets, presentations and such?
The main issue I see is that the use case for designers is definitely there.. Inkscape f.e. is a wonderful tool; once you get the hang of it for creating single-page designs, sooner or later, it is highly likely that you would want to create a multi-page document. And being forced to create a single page per file, then glue the exported PDFs together is an annoying workflow complication.
Now assuming inkscape integrates some solution (as in the aforementioned plug-in, c.f. [inkscape-pages-1.2.1/modules/pages.py](https://github.com/w3c/svgwg/files/324630/pages.py.txt)) to this into core, while the issue would be solved for inkscape users, the exchange of these multi-page SVG documents with other applications would then become the next issue. So while SVG primarily is a standard for the web, maybe it can still accommodate to this most basic need of DTP/print?
> Support for multi-page "documents" is out-of-scope for SVG

Is that the common agreement amongst the SVG consortium?

**Comment by liamquin** [comment]
On Tue, 2016-06-14 at 08:08 -0700, Marcel Partap wrote:
> 
> From a user standpoint, this is mainly useful in creating paginated
> documents for print

There's a related area (or, I think it's related) of identifying parts
of an image as separate, for making e.g. responsive & accessible
infographics. For example, an image that on a desktop formats as

    A => B => C
might on a mbile device (or in a narrow window) be better as 

    A
    ⇓
    B
    ⇓
    C

But maybe that's better done with making A, B, C separate images? I'm
not sure.

**Comment by karlcow** [comment]
I propose we are closing this as out of scope. Multi-page document support is better addressed by CSS Paged Media or other document formats. SVG is focused on scalable vector graphics within a single viewport.

**Comment by dirkschulze** [comment]
RESOLUTION: Multi pages will not get adapted in SVG2

https://www.w3.org/2026/03/26-svg-minutes.html

**Comment by karlcow** [comment]
Should we close the issue.
